core/src/main/java/tachyon/client/TachyonFS.java

```diff
@@ -552,9 +555,8 @@ public synchronized TachyonFile getFile(TachyonURI path, boolean useCachedMetada
    * 
    * @param path the path in Tachyon file system
    * @return the file id if exists, -1 otherwise
-   * @throws IOException
    */
-  public synchronized int getFileId(TachyonURI path) throws IOException {
+  public synchronized int getFileId(TachyonURI path) {
```

Comment:
when calling getFileId, we hope it will return an integer, this method is often used like function(getFileId(path)), so return -1 on error looks like a nice solution both semantically and practically.